ajax/libs/annyang/package.json

```diff
@@ -22,10 +22,9 @@
   "autoupdate": {
     "source": "git",
     "target": "git://github.com/TalAter/annyang.git",
-    "basePath": "",
+    "basePath": "dist/",
     "files": [
-      "annyang.min.js",
-      "annyang.js"
+      "*"
```

Comment:
@TalAter I'll like to use `**/*` here so that it can also fetch other files under each directories if there are directories in future, what do you think?